lib/cretonne/meta/isa/intel/registers.py

```diff
@@ -46,7 +46,9 @@
         names=['eflags'])
 
 GPR = RegClass(IntRegs)
+GPR_NORIP = GPR.without(GPR.rbp, GPR.r13)
```

Comment:
Minor bikeshed: How about `GPR_NO_RBP_R13` :-).

Alternatively, could you add a brief comment here explaining the "NORIP" name? Something like "In some contexts, the rbp/r13 encoding is interpreted to mean rip" or so.

Reply:
done!